src/components/publishing/sections/caption.tsx

```diff
@@ -18,7 +18,7 @@ interface FigcaptionProps {
 }
 const div: StyledFunction<FigcaptionProps & React.HTMLProps<HTMLDivElement>> = styled.div
 const Figcaption = div`
-  & > p, p {
+  & > p, p, .public-DraftEditorPlaceholder-root {
```

Comment:
It may be a good idea to add a comment that explains what this class is for so it doesn’t accidentally get removed in the future.